stdlib/public/TensorFlow/Ops.swift

```diff
@@ -818,7 +818,7 @@ public extension Tensor where Scalar == Bool {
   @inlinable @inline(__always)
   func all() -> Bool {
     let axes = Tensor<Int32>(rangeFrom: 0, to: rank, stride: 1)
-    return Raw.all(self, reductionIndices: axes).scalarized()
+    return _TFGetScalarOrDie(Raw.all(self, reductionIndices: axes))
```

Comment:
would this trigger sends/recvs because of https://github.com/apple/swift/blob/2b0fbd0be430cbcd89652e431c521a967f8daea3/stdlib/public/TensorFlow/DataTypes.swift#L72?

Reply:
No. TFDeabstraction or TFPartition has been taught to replace the force unwrapping (`.scalar!`) with the tensor itself, when the consumer of the value is `_TFMakeScalarTensor(_:)`. In other words:

```swift
%y = _TFGetScalarOrDie(%x)
...
%z = _TFMakeScalarTensor(%y)
```
... will just become `%x`

In fact, `.scalarized()`'s implementation is just `_TFGetScalarOrDie(reshaped(to: []).handle)`. We are just eliminating an unnecessary reshape here.

Reply:
Great, thanks for clarifying!